lightning/src/ln/channelmanager.rs

```diff
@@ -210,6 +212,33 @@ pub struct PaymentPreimage(pub [u8;32]);
 #[derive(Hash, Copy, Clone, PartialEq, Eq, Debug)]
 pub struct PaymentSecret(pub [u8;32]);
```

Comment:
Perhaps we can move `PaymentSecret` into a more low-level module like `ln`. We can reexport it in `channelmanager` to avoid making any cascading changes, if needed.

Reply:
Feels a bit odd to separate out the PaymentSecret from the PaymentPreimage/PaymentHash. Maybe they should all be moved in follow-up?

Reply:
I mean its a few extra lines of diff, might as well move them all in one go.